beamer/tests/agent/unit/test_handlers.py

```diff
@@ -25,6 +26,28 @@
 from beamer.tests.agent.utils import make_address
 from beamer.tests.constants import FILL_ID
 from beamer.typing import FillId, Termination
+from web3.datastructures import AttributeDict
+from typing import cast
+
+
+def get_tx_receipt(status, tx_hash) -> TxReceipt:
+    data = {
+            "status": status,
+            "transactionHash": tx_hash,
+            "blockHash": HexBytes(0),
+            "blockNumber": BlockNumber(1),
+            "contractAddress": None,
+            "cumulativeGasUsed": 0,
+            "effectiveGasPrice": 1,
+            "gasUsed": Wei(1),
+            "from": ADDRESS1,
+            "logs": [],
+            "logsBloom": HexBytes(0),
+            "root": HexStr("1"),
+            "to": ADDRESS1,
+            "transactionIndex": 1,
+        }
+    return cast(TxReceipt, AttributeDict(data))
```

Comment:
I assume `AttributeDict` supports kwargs so instead of first creating a dict and then copying it to make another one, why not just create `AttributeDict` directly, with kwargs?

Reply:
from is making the problem :(

Reply:
and AttributeDict needs dict so kwargs are not working.

Reply:
:woozy_face: